### DISCUSSION TITLE
Stay Up to Date with News on .NET Previews

### DISCUSSION THREAD

**Discussion by jamesmontemagno** [dotnet/core#9192] [News]
We just [released our first preview of .NET 9](https://github.com/dotnet/core/discussions/9167) and you may have noticed that we published the announcement on GitHub Discussions instead of the .NET Blog. We also made announcements for other features & projects in their repos Discussions including .NET MAUI, ASP.NET Core, EF Core, and more. Each month we will be making announcements on GitHub Discussions for each preview. We'll tailor our .NET blog content to highlight the advantages of .NET 8, aiming to support your use of .NET 8 in production environments. You can read more about our strategy in our [announcement post](https://github.com/dotnet/core/discussions/9131).

So, a question you may have is, how do I stay up to date?

There are several ways based on how involved and how many notifications you would like to receive.

## Minimal Notifications

The [dotnet/announcements](https://github.com/dotnet/announcements) repo is intended to keep everyone informed about important information related to the .NET platform. It is a one-way broadcast that plays nicely with GitHub notifications, making it desirable to watch/subscribe to it. By watching the repo and subscribing to **All Activity** you will get minimal alerts including one for each preview, monthly service releases, and CVEs.

Here is the [.NET 9 Preview 1 announcement](https://github.com/dotnet/announcements/issues/297) on dotnet/announcements.

Read our [best practices guide](https://github.com/dotnet/core/issues/7562) to learn more.

## RSS Feeds

GitHub Discussions over atom feeds that you can subscribe to that are updated when new posts are made. You can subscribe to all discussions in all categories or specific categories.

All Discussions: https://github.com/dotnet/core/discussions.atom
Only News: https://github.com/dotnet/core/discussions/categories/news.atom

This can be applied to any repo, for example if you want to subscribe to ASP.NET Core announcements you would use:

https://github.com/dotnet/aspnetcore/discussions/categories/announcements.atom

## GitHub Notifications for All Discussions & Comments

Want to be super involved and get notified for all new discussions and responses as well? You can turn on custom notifications and select *Discussions** and you will get a notification in GitHub whenever there is a new discussion thread and comments on it.

**Comment by iamcraz** [comment]
is it have real diffrence from v8 ??

**Comment by MikeAndrews90** [comment]
"Subscribing to All Activity you will get minimal alerts" - Yeah... That hasn't worked out at all. I'm getting an email for every comment. Time to unsubscribe 💀

**Reply by jamesmontemagno** [reply]
As noted in my post the https://github.com/dotnet/announcements is ONLY read-only announcements and not discussions, this would be the repo to subscribe to if you want minimal announcement. Not this repo or discussions based on how GitHub notifications work

**Reply by MikeAndrews90** [reply]
Apologies, thanks.

**Comment by loic-sharma** [comment]
Most of the https://github.com/dotnet/announcements announcements are security announcements. Would it make sense to move .NET security announcements to a separate repository, perhaps `dotnet/security` or `dotnet/announcements-security`?

**Reply by jamesmontemagno** [reply]
@richlander and I were discussing that and other ideas to help collapse some of these things down into a single monthly post in /announcements. We weren't thinking of CVEs, but they could be collapsed down as well. We will keep ya posted.

**Comment by Securso** [comment]
Can someone point me to a simple list of what is appearing in the next preview of .Net 9. Just like you used to do. I'm a retired developer who is now writing open source software using the latest releases of .Net. I don't want to go wading through announcements and core repo discussions.  I find this new system very confusing. I guess I now have to wait for preview 2 release notes.

**Reply by jamesmontemagno** [reply]
Release notes for each preview are found here in the repo. Here are preview 1: https://github.com/dotnet/core/tree/main/release-notes/9.0/preview/preview1

Preview 2 will be in a preview 2 folder. 

Each product also has a what's new doc which is kept up to date as each preview is released, for example: https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/overview
